pandas/core/series.py

```diff
@@ -1586,6 +1586,58 @@ def __repr__(self):
 
         return result
 
+    def _repr_html_(self):
+        # TODO: Full independent HTML generation in SeriesFormatter, rather
+        # than depending on a limited subset of functionality via to_frame().
+
+        if get_option("display.notebook_repr_html"):
+            max_rows = get_option("display.max_rows")
+            min_rows = get_option("display.min_rows")
+            max_cols = get_option("display.max_columns")
+            show_dimensions = get_option("display.show_dimensions")
+
+            formatter = fmt.DataFrameFormatter(
```

Comment:
Is there a particular reason for specifying all of the keyword arguments here?

Reply:
I'm just copying the behaviour from `DataFrame._repr_html`, which also specifies all the arguments. I assumed this was to decouple the method from any changes to default values in `to_html`.

Looking at it again, `max_cols` could be removed as it isn't relevant to a series.

Reply:
Have removed any references to the `max_cols` config item now. I've left all the other defaults in to be consistent with the `DataFrame._repr_html_()` behaviour, please let me know if you want me to deviate from this.